setup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+from setuptools import setup
+
+with open("README.md", 'r') as f:
+    long_description = f.read()
+
+with open("requirements.txt", 'r') as f:
+    requirements = [i.strip() for i in f.readlines()]
+
+setup(
+    name="xCell",
+    version="dev",
+    description=("Pipeline to compute the Nx2pt angular power spectra and their covariances."),
+    license="GPL3",
```

Comment:
Did we add this license file? Also, I think we shouldn't use GPL3 because it's more restrictive than BSD3 (e.g. https://github.com/LSSTDESC/NaMaster/blob/master/LICENSE)

Reply:
Yep. At the beginning of time. I will not make much deal about it but I would keep it GPL because it's more restrictive: I think it's a good thing forcing others not to enclose their code if they use public libraries.

Reply:
It spreads like a virus. There may be reasons why people want to use other free software licenses (e.g. because their code has applications in the industry and otherwise it won't be used). As an example, the use of GPL3 by healpix makes it a nightmare for us to use it as a dependency in namaster.

I am fine with GPL, by the way, just not GPL3 (GPL2 is fine)

Reply:
That constraint is only there if you use code in your own code, not if you use it as an external dependency (i.e. if we had copy-pasted parts of healpy into our code, or if we were compiling healpy as part of our code, or if we were writing a modified version of healpy)

Reply:
if you like GPL let's do GPL2